src/config/plugins/pluginData.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "pluginInstallStatus": false,
+    "pluginName": "Posts",
+    "pluginCreatedBy": "Talawa Team",
+    "pluginDesc": "Allow users to create, comment and like and share the content in the form of Pictures and Videos.",
+    "installedOrgs": ["62ccfccd3eb7fd2a30f41601", "62ccfccd3eb7fd2a30f41601"]
+  },
+  {
+    "pluginInstallStatus": false,
+    "pluginName": "Events22222222",
+    "pluginCreatedBy": "Talawa Team",
+    "pluginDesc": "Allow users to register and attend for new events with a inbuilt calendar to keep track of daily events.",
```

Comment:
We already have a directory called `sample_data` in the root of our project that hosts all the sample data that the developers can load. Please move this file there too. Also, ensure that the organization and user IDs used here are compatible with the user IDs used in that sample data.

Reply:
Yes eshan, I'm aware about that, actually for now I just wanted to keep my plugin logic separate.
And I think the data in the sample data folder is for creating a sample demo app so that user can test the features but that may not be used in production mode.
So, that's why I've kept it in a separate place.